src/components/form/form.scss

```diff
@@ -136,3 +136,7 @@
     padding-left: 1rem;
     padding-top: 0.25rem;
 }
+
+.add-new {
```

Comment:
lime-elements should know nothing about Lime Admin. We might still need to add this styling here, but it should probably be called something that's relevant and understandable to any consumer of lime-elements.

(It's a good hint that something is wrong when the title of a PR in lime-elements references a specific consumer, like this one does. The change should preferably be possible to explain and justify in more general terms. Then we can of course always mention that this will be used in Lime Admin.)

Reply:
Sorry I didn't understand the problem. 

<img width="50%" alt="Screenshot 2021-11-17 at 12 08 26" src="https://user-images.githubusercontent.com/35954987/142189699-5a48103f-fa42-479e-a2bf-061cd8b38abc.png">
The button is already used in forms in the admin, also in the examples in Elements. I'm just changing the styles. 

Do you mean I should change the title of the branch? For the change log, isn't it enough that the commit titles are explanatory? Or what do you mean?

Reply:
I mean that the class probably shouldn't be called `add-new`.

Also, come to think of it, I think this is solving only a very specific instance of a more general problem. I think the only reason that the text-fields higher up in that screenshot has nice margins is that that's where the helper-text would go if they had any. We've got an issue somewhere about selects needing some margin in Lime Admin, precisely because the helper-text overlaps the label of the next field.

In short, I think we simply need to set a `gap`…

Reply:
> I mean that the class probably shouldn't be called add-new.

I see. Do you have any suggestions? I don't really know what to call it to be generic. `form-field-button`? 

> In short, I think we simply need to set a gap…

Yes, we made an issue here: https://github.com/Lundalogik/crm-feature/issues/2466 and talked with @jgroth who points out that solving this for Admin seems to be a bit tricky, using a single CSS `@prop`. We would need to find solutions so that we can write it like the way we do for columns, 

e.g.
````py
…
lime {
	'layout' : {
        'type': 'grid',
        'column': 3,
        'gap': 2,
    }
}
````